src/ClientHandle.cpp

```diff
@@ -2320,6 +2320,15 @@ void cClientHandle::SendHealth(void)
 
 
 
+void cClientHandle::SendHideTitle(void)
```

Comment:
There is a slight possibility that you'd want to display the subtitle, but hide the main title, that's why a separate method is still a good idea.